lib/controller.js

```diff
@@ -744,7 +744,7 @@ controller.root = function(opts) {
       // Spawn a new SSH process
       var child = cp.spawn('ssh', ['-i',
         // Use the provided key path
-        opts.key,
+        Tessel.LOCAL_AUTH_KEY,
         // Connect to the Tessel's IP Address
```

Comment:
@Student007 in my opinion, it's the best solution because it's the least amount of changed code.